tests/parser/ScheduleTests.cpp

```diff
@@ -5071,3 +5069,61 @@ PERMZ
         BOOST_CHECK_THROW(sched_grid.get_cell(2,2,2), std::exception);
     }
 }
+
+BOOST_AUTO_TEST_CASE(Test_wvfpexp) {
+        std::string input = R"(
+        DIMENS
```

Comment:
Tiny nit: These lines (`DIMENS` &c) should really start at the left margin.  Otherwise, it'd be as if the input file were written with leading spaces as
```
        DIMENS
          10 10 10 /
        -- ...
```
instead of typical
```
DIMENS
  10 10 10 /
-- ...
```
The opm-common input parser currently happens to be able to read these such files, but may or may not keep that ability in the future.  I think it's better to stick strictly to the expected format of no leading whitespace characters on keyword lines for our regression tests.